src/main/java/eu/europeana/downloads/ListRecordsExecutor.java

```diff
@@ -1,73 +0,0 @@
-package eu.europeana.downloads;
-
-import org.apache.logging.log4j.LogManager;
-import org.apache.logging.log4j.Logger;
-
-import java.util.List;
-import java.util.concurrent.Callable;
-
-public class ListRecordsExecutor implements Callable<ListRecordsResult> {
-
```

Comment:
I guess this isn't used by the ListRecords method?

Reply:
Yes, this is not used by ListRecord. the naming was quite confusing. hence I created a new class ListIdentifierExecutor